src/Php54/Php54.php

```diff
@@ -39,4 +39,63 @@ public static function hex2bin($data)
 
         return $data;
     }
+
+    public static function http_response_code($response_code)
+    {
+        static $current_code = 200;
+
+        $messages = array(
```

Comment:
The function needs to return its previous value, so we need a static variable for this.
Are you saying that you would prefer a class-level static, rather than a function-level static?